app/controllers/merchant/items_controller.rb

```diff
@@ -1,7 +1,50 @@
 class Merchant::ItemsController < Merchant::BaseController
 
+  def new
+    @merchant = Merchant.find(current_user.merchant_id)
+    @item = @merchant.items.new
```

Comment:
Do you need to pass in 2 variables here?  You should be able to access items through @merchant.items.  Mike told me to avoid this kinda thing if you can.

Reply:
Updated this and the `new.html.erb` view to use only `@item` variable

Reply:
Thanks for fixing this!